src/test/resources/karate/DissolutionController.feature

```diff
@@ -0,0 +1,5 @@
+Scenario: Check if application is up
+
+    Given url 'http://localhost:9000/dissolution/healthcheck'
```

Comment:
9000? how? Were you able to run this?

Reply:
These are some obsolete tests that I did before handling off testing to Szymon. Will remove that.